### ISSUE TITLE
hieradata directory in PE 2015.2 multi-environment

### ISSUE THREAD

**Issue by dan-wittenberg** [hunner/puppet-hiera#83]
It appears you can't leave datadir blank, and you can't ignore that setting, it always wants to write something to hiera.yaml.  On PE 2015.2 it's blank by default so multi-environment works, but it breaks after this because hiera is no longer looking in the environment, only in one location.  There should be an option to leave it blank like it is by default.

**Comment by dogfish182** [comment]
I noticed the same. I just copied the line in the comments of a new installed hiera yaml and specified that as my datadir. But I agree, it should be able to NOT set it. if you attempt not to set it, it breaks your multi-environ hierasetup.

**Comment by dan-wittenberg** [comment]
Exactly.  I think if you put nothing in there then it should just add the variable with no value.
